backend/dal/Migrations/Initial/PostDeploy/01-ViewProperties.sql

```diff
@@ -1,79 +1,92 @@
+--DROP VIEW dbo.View_Properties
 -- This script provides a way to union both parcels and buildings into a single result of properties.
-CREATE VIEW View_Properties AS
+CREATE VIEW dbo.[View_Properties] AS
```

Comment:
Updating view to include additional information.  You can manually apply the view to your DB if you don't want to refresh.